2017/JAN/LTIME44/NOTINCOM/NOTINCOM.cpp

```diff
@@ -0,0 +1,28 @@
+// Code by Piyush
+#include <bits/stdc++.h>
+#define endl "\n"
+using namespace std;
+
+int main() {
+
+    std::ios::sync_with_stdio(false);
+    int testCases;
+    cin >> testCases;
+    while (testCases--) {
+        int n, m;
+        cin >> n >> m;
+        vector<short int> collection(1e+6, 0);
+        for (int i = 0; i < n + m; i++) {
```

Comment:
Try to use ++i instead of i++, as this repo is concerned towards competitve programming. So the solution should be fast and efficient in terms of solution.

Follow this [link](https://stackoverflow.com/questions/24901/is-there-a-performance-difference-between-i-and-i-in-c).

Reply:
It doesn't make major difference for primitives

Have a look at [this](https://www.quora.com/Whats-the-difference-between-i++-and-++i-as-for-a-loop-increment-statement-in-C++)

Reply:
Also, resolve conflicts ASAP.